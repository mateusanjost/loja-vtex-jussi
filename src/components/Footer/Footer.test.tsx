import TestRenderer from "react-test-renderer";
import FooterComponent from "./index";

describe("Footer", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<FooterComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
