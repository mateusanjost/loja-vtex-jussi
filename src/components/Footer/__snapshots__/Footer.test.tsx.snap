// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer
  className="sc-bdnxRM vwvka"
>
  <div
    className="sc-gtsrHT fFwKqr"
  >
    <img
      alt="WPP Company logo"
      src="wppcompany1x.svg"
      srcSet="wppcompany1x.svg, wppcompany2x.svg 2x"
    />
  </div>
  <div
    className="sc-dlnjwi TuFfO"
  >
    <a
      href="https://www.facebook.com/agencia.jussi/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Facebook Logo"
        src="facebook1x.svg"
        srcSet="facebook1x.svg, facebook2x.svg 2x"
      />
    </a>
    <a
      href="https://www.instagram.com/jussi/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Instagram Logo"
        src="instagram1x.svg"
        srcSet="instagram1x.svg, instagram2x.svg 2x"
      />
    </a>
    <a
      href="https://www.linkedin.com/company/agencia-jussi"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Linkedin Company Logo"
        src="linkedin1x.svg"
        srcSet="linkedin1x.svg, linkedin2x.svg 2x"
      />
    </a>
  </div>
</footer>
`;
